libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/VerifiableCredential.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.di.ipv.core.library.domain;
+
+import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.SignedJWT;
+import lombok.Getter;
+import uk.gov.di.ipv.core.library.exceptions.CredentialParseException;
+import uk.gov.di.ipv.core.library.persistence.item.VcStoreItem;
+
+import java.text.ParseException;
+
+@Getter
+public class VerifiableCredential {
+    private final String userId;
+    private final String criId;
+    private final String vcString;
+    private final JWTClaimsSet claimsSet;
+
+    // Only construct after validation
+    public VerifiableCredential(String userId, String criId, SignedJWT jwt)
```

Comment:
If you want to make this more explicit, you could make the constructor private and add another static factory method `fromValidJwt` (or similar).

Reply:
That's good I'll do that!